### DISCUSSION TITLE
Repo

### DISCUSSION THREAD

**Discussion by FelixHennerich** [community/community#62611] [Repositories]
### Select Topic Area

Question

### Body

How do i open a repository?
Best Regards :))

**Comment by Caaasperrr** [comment]
Hi, 
go to github.com then navigate to your profile picture in the top right. Go to "your repository". At last u have to press the green new button. I hope it helps you :))

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
